etc/behat/services/contexts.xml

```diff
@@ -46,6 +46,7 @@
         <service id="sylius.behat.context.setup.geographical" class="%sylius.behat.context.setup.geographical.class%" scope="scenario">
             <argument type="service" id="sylius.factory.country" container="symfony" />
             <argument type="service" id="sylius.repository.country" container="symfony" />
+            <argument type="service" id="sylius.behat.converter.country_name_to_code" container="symfony" />
```

Comment:
The service is defined in default, "behat", container, it should throw an exception if trying to get it from symfony container.